manifests/P/PlasmoApp/plasmo_voice/main.yaml

```diff
@@ -0,0 +1,23 @@
+manifestSpecVersion: "4.0"
+publisher: PlasmoApp
```

Comment:
The publisher seems to be `plasmoapp`, not `PlasmoApp`. Any reason you chose the second one?